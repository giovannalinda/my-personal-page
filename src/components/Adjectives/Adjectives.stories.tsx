import { Meta, Story } from '@storybook/react'
import { Adjectives } from './Adjectives'

export default {
  title: 'Components/Adjectives',
  component: Adjectives,
} as Meta

export const Default: Story = () => <Adjectives />
