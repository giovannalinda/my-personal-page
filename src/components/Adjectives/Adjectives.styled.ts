import styled from 'styled-components'
import { theme } from '~/styles'

export const List = styled.ul`
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  margin-bottom: 40rem;
  margin-top: 40rem;
`

export const ListItem = styled.li`
  padding-bottom: ${theme.space[7]};

  &:hover {
    color: ${theme.colors.purple};
  }
`

export const Title = styled.h1`
  text-decoration: underline;
  font-size: ${theme.font.sizes.title};
  transition: ${theme.transitions.slow};

  @media screen and (max-width: ${theme.breakpoints.smallMobile}) {
    font-size: ${theme.font.sizes.large};
  }
`
