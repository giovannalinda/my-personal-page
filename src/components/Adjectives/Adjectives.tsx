import { useTranslation } from '~/lib'

import * as S from './Adjectives.styled'

export function Adjectives() {
  const { t } = useTranslation()

  return (
    <S.List>
      <S.ListItem>
        <S.Title>{t('Creativity')}</S.Title>
      </S.ListItem>
      <S.ListItem>
        <S.Title>{t('Singularity')}</S.Title>
      </S.ListItem>
      <S.ListItem>
        <S.Title>{t('Curiosity')}</S.Title>
      </S.ListItem>
    </S.List>
  )
}
