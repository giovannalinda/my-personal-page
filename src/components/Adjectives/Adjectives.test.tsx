import { render } from '~/__helpers__/app-tests'
import { Adjectives } from './Adjectives'

describe('Adjectives', () => {
  it('matches snapshot', () => {
    const { container } = render(<Adjectives />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
