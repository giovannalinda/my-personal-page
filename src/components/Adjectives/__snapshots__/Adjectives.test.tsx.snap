// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Adjectives matches snapshot 1`] = `
<ul
  class="sc-bczRLJ jksxgQ"
>
  <li
    class="sc-gsnTZi kOgEjT"
  >
    <h1
      class="sc-dkzDqf lmAeTQ"
    >
      Creativity
    </h1>
  </li>
  <li
    class="sc-gsnTZi kOgEjT"
  >
    <h1
      class="sc-dkzDqf lmAeTQ"
    >
      Singularity
    </h1>
  </li>
  <li
    class="sc-gsnTZi kOgEjT"
  >
    <h1
      class="sc-dkzDqf lmAeTQ"
    >
      Curiosity
    </h1>
  </li>
</ul>
`;
